extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -146,62 +146,47 @@ describe.configure().retryOnSaucelabs().run('amp-bind', function() {
     });
   });
 
+  /**
+   * Combined test specs for amp-img
+   */
   describe('amp-img integration', () => {
-    it('should change src when the src attribute binding changes', () => {
-      const button = fixture.doc.getElementById('changeImgSrcButton');
+    it('should do amp-img integration', () => {
+      const changeImgSrcButton = fixture.doc
+          .getElementById('changeImgSrcButton');
+      const invalidSrcButton = fixture.doc.getElementById('invalidSrcButton');
+      const ftpSrcButton = fixture.doc.getElementById('ftpSrcButton');
+      const changeImgAltButton = fixture.doc
+          .getElementById('changeImgAltButton');
+      const changeImgDimensButton = fixture.doc
+          .getElementById('changeImgDimensButton');
       const img = fixture.doc.getElementById('image');
       expect(img.getAttribute('src')).to.equal('http://www.google.com/image1');
-      button.click();
+      expect(img.getAttribute('height')).to.equal('200');
+      expect(img.getAttribute('width')).to.equal('200');
+      changeImgSrcButton.click();
+      invalidSrcButton.click();
+      ftpSrcButton.click();
+      changeImgAltButton.click();
+      changeImgDimensButton.click();
```

Comment:
What would be really cool here would be combining the actions that we do expect to happen into one button. So rather than having three buttons that each change `src`, `alt`, or `width` and `height`, you have one button that changes all three at once. We would still want to keep the other ones `invalidSrcButton` and `ftpSrcButton` distinct since we expect them not to change the `src` attribute.